spec/datatables/user_datatable_spec.rb

```diff
@@ -90,9 +90,9 @@
     it { is_expected.to respond_to(:view_columns) }
   end
 
-  context 'outputs' do
+  skip 'outputs' do
```

Comment:
I would rather this context have all the tests within it pending instead of immediately skipping the entire block of tests